packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -289,24 +290,30 @@ Future<T> showModalBottomSheet<T>({
 
 /// Shows a persistent material design bottom sheet in the nearest [Scaffold].
 ///
+/// Returns a controller that can be used to close and otherwise manipulate the
+/// bottom sheet.
+///
+/// To rebuild the bottom sheet (e.g. if it is stateful), call
+/// [PersistentBottomSheetController.setState] on the controller returned by
+/// this method.
+///
+/// The new bottom sheet becomes a [LocalHistoryEntry] for the enclosing
+/// [ModalRoute] and a back button is added to the appbar of the [Scaffold]
+/// that closes the bottom sheet.
```

Comment:
encloses?

btw is this behaviour of the appbar expected? i'm theoretically working on a patch that might break that.

Reply:
As far as I can tell from https://material.io/design/components/sheets-bottom.html#standard-bottom-sheet, persistent bottom sheets don't behave like routes viz the back button.

I'm fine with collapsing the two types of persistent bottom sheets and leaving the LocalHistoryEntry back button stuff behind. It would be an incompatible change of course.